src/main/java/frc/robot/subsystems/intake/BeamBreaker.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+
+public class BeamBreaker extends LoggedSubsystem<BeamBreakerLoggedInputs> {
+    public DigitalInput beam = new DigitalInput(Ports.Intake.BEAMBREAKER);
+    boolean isBlocking;
+
+    public BeamBreaker(BeamBreakerLoggedInputs inputs) {
+        super(inputs);
+    }
+
+
+    public boolean getBeam() {
```

Comment:
1. Boolean symbols (variables, functions, etc.) so start with "is", "has", "should", etc.
2. This function doesn't return a "beam", so it shouldn't be called `getBeam`.
3. The "beam" is an implementation detail. The name should be suited for end-users, i.e., "does the intake hold a game piece?".